include/oneapi/dpl/pstl/glue_memory_impl.h

```diff
@@ -226,14 +228,23 @@ destroy(_ExecutionPolicy&& __exec, _ForwardIterator __first, _ForwardIterator __
 
     const auto __is_parallel =
         oneapi::dpl::__internal::__is_parallelization_preferred<_ExecutionPolicy, _ForwardIterator>(__exec);
-    const auto __is_vector =
-        oneapi::dpl::__internal::__is_vectorization_preferred<_ExecutionPolicy, _ForwardIterator>(__exec);
+    using _is_vector_type =
+#if _ONEDPL_ICPX_OMP_SIMD_DESTROY_WINDOWS_BROKEN
```

Comment:
Sorry, I missed one more comment.
Every where in the current ondDPL code in the simular cases we check two macroses: 
 _ONEDPL_* and _PSTL_*..
 So, we have to add  _PSTL_ICPX_OMP_SIMD_DESTROY_WINDOWS_BROKEN everywhere below.

Reply:
Thanks for noticing it. It has been fixed in #1195.